code/game/atoms_movable.dm

```diff
@@ -126,6 +126,10 @@
 	/// Example: If req_one_access = list(ACCESS_ENGINE, ACCESS_CE)- then the user must have either ACCESS_ENGINE or ACCESS_CE in order to use the object.
 	var/list/req_one_access
 
+	/// if truthy, rcd spritesheets will use this as the key to this movable's cached icon
+	/// instead of its name.
+	var/rcd_spritesheet_override = ""
```

Comment:
this is REAL weird, you override it for two cases what are they targeting exactly?

Reply:
full tile windows are the default but aren't called "windows" on master because of inheritance from a time when directional windows were the default, leading to a situation where 99% of windows are clumsily specified as "full tile windows". wallening fixed this, but it caused issues with rcd spritesheets where the names of full tile and directional windows overlapped and thus used the same icons in the menu. So either 1. we revert it and make full tile windows be named "full tile windows", 2. we name directional windows "directional windows" which is still a mouthful for something you can visually identify instantly but would also require overriding the name on every full tile window, or 3. override the spritesheet key to avoid forcing restrictions on player facing info based on technical limitations

i picked 3

Reply:
could you write this down somewhere (maybe in the override?)